api/src/opentrons/protocol_engine/types.py

```diff
@@ -398,3 +407,17 @@ class HeaterShakerMovementRestrictors:
     plate_shaking: bool
     latch_closed: bool
     deck_slot: int
+
+
+class CalibrationPosition(str, Enum):
+    """Deck slot to move to."""
+
+    PROBE_POSITION = "probePosition"
+    ATTACH_OR_DETACH = "attachOrDetach"
```

Comment:
Let's rename these to be more consistent while we're here moving stuff around:

- If the enum is called `CalibrationPosition`, the value shouldn't have "position" in its name too; it's redundant
- Gripper attach location may be different than the pipette attach position

I'd suggest:

```suggestion
    PIPETTE_PROBE_ATTACH = "pipetteProbeAttach"
    PIPETTE_ATTACH = "pipetteAttach"
```

This leaves space for both `GRIPPER_PROBE_ATTACH` and `GRIPPER_ATTACH` if we need them. I don't think it's really necessary to spell out "attach or detach" here; if you're detaching, it's likely to attach a new pipette, so it's basically the same thing

Reply:
Made sure with Laura and Linda that the position for attach/detach a gripper is the same as the pipette.

Reply:
`INSTRUMENT_PROBE_ATTACH` seems fine to me.